pkg/apis/jaegertracing/v1/const.go

```diff
@@ -0,0 +1,9 @@
+package v1
+
+const (
+	// LabelManagedBy is used as the key to the label indicating that this instance is managed by an operator
+	LabelManagedBy string = "app.kubernetes.io/managed-by"
```

Comment:
Although I don't dislike the use of this k8s label, its description is "The tool being used to manage the operation of an application", with example `helm`. Don't think it would cause problems using it, but wanted to raise the point before we commit to it.

Reply:
Thought about it as well yesterday when I read the doc on it again, but I think it does apply: the operator is managing the operation of the Jaeger instance. 

For the underlying objects (deployment, services, ...), we set this value to `jaeger-operator`, but we might also want to consider changing it to use *which* operator instance in the future.

In any case, I'll ask around and see if there's something more appropriate.

Reply:
Addressed by #576